cpt.yaml

```diff
@@ -127,3 +119,27 @@ tools:
   - name: cpt_fasta_translate
     owner: cpt
     tool_panel_section_label: Annotation
+    
+  - name: cpt_protein_blast_grouping
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: cpt_prophage_relatedness
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: cpt_intersect_adj
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: phageterm
+    owner: mmonot
```

Comment:
We need to review those tools before we can merge them.

Reply:
Can you point me maybe to the github repo of those tools?

Reply:
https://github.com/avilella/phageterm  <- is a fork 

The only reference on github is https://github.com/TAMU-CPT/galaxy-tools/blob/c43f5a7706fb64615b7c050a5a9e5fc20ca33873/tool_conf.xml.old#L76 ... any reason you have hidden it on your instance?


I think I found the latest version https://gitlab.pasteur.fr/vlegrand/ptv/-/tree/master

But the Galaxy wrapper is 6 years old and has not been updated. I could not find the source on github, the source in the TS is insecure and needs updates.